lib/generators/templates/devise.rb

```diff
@@ -135,6 +135,11 @@
   # unconfirmed_email column, and copied to email column on successful confirmation.
   config.reconfirmable = true
 
+  #If true, requires any email changes to be confirmed (exactly the same way as
+  # initial account confirmation) to be applied.
+
+  config.new_email_confirmable = false
```

Comment:
This template should use the `send_reconfirmation_instructions` from `lib/devise.rb` :)